pkg/config/yaml.go

```diff
@@ -74,6 +74,26 @@ func importBlueprint(f string) (Blueprint, YamlCtx, error) {
 	return bp, yamlCtx, nil
 }
 
+func importDeploymentFile(f string) (DeploymentSettings, YamlCtx, error) {
```

Comment:
`importBlueprint` has tests in `config_test.go`. `importDeploymentFile` should also have a test.

Reply:
I added a test for the public function that covers various failure methods of the internal private function (empty YAML, YAML with unknown fields).  I added these tests for importBlueprint while I was at it. Overall coverage up 0.1%.